volmdlr/faces.py

```diff
@@ -4686,9 +4686,10 @@ def _data_hash(self):
     def _data_eq(self, other_):
         if other_.__class__.__name__ != self.__class__.__name__:
             return False
-        self_set = set([self.point1, self.point2, self.point3])
-        other_set = set([other_.point1, other_.point2, other_.point3])
-        if self_set != other_set:
```

Comment:
I mean self_set != other_set ?

Reply:
Because i had an error with set method if points are Node3D. Indeed, a list on Node3D seemed to be rearranged whereas Point3D list is not.
Perhaps we can let this PR on the side because i did not have an error since this moment.